langchain/src/llms/portkey.ts

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable no-process-env */
+import _ from "lodash";
```

Comment:
We don't have this as a dependency

Reply:
Added Lodash as dependency. Please let me know if that is something that can be added a dependency or should it be placed under devDependencies.

Reply:
It's just being used here:

https://github.com/langchain-ai/langchainjs/pull/2763/files#diff-c81633c54fc433c468eca977ec6396e374fa3fdb9aacc67eb5f6e63d983241abR55

If you really must, you can add it as a `devDependency` and a `peerDependency` and require people to separately install it to use Portkey.

Reply:
Since it's not something that is foreseen to be used anywhere in the package, I will move this to the `devDependency`